english/ninja/ninja.md

```diff
@@ -16,7 +16,7 @@ You'll get what you ask for
 I'm like a wild animal in the corner
 Waiting for the break of dawn
 Trying to get through the night
-Just a man with the will to survive
+Just a man with the will to [survive](../pinch.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.